Automattic/newspack-plugin
# Onboarding: Homepage patterns

### All Submissions:






### Changes proposed in this Pull Request:



Adds homepage pattern chooser to onboarding flow.

### How to test the changes in this Pull Request:

1. In Design Wizard, confirm there is **no** homepage patterns chooser above the Typography section
2. On a site with Newsletters & Reader Revenue configured, visit the Design step of the onboarding, observe that there's a homepage pattern chooser above the Typography section
2. Test all patterns by selecting each and triggering preview using the button at the bottom of the screen
3. All patterns should contain a donation section, and the first pattern also a subscribe section
3. Set up the plugin on a new site - in the Services step of the onboarding, do not configure newsletters and reader revenue
4. Test the patterns - none should contain the donate section nor the subscribe section

### Other information:
